ee/maintained-apps/ingesters/winget/ingester.go

```diff
@@ -306,6 +310,8 @@ func (i *wingetIngester) ingestOne(ctx context.Context, input inputApp) (*mainta
 	if input.UniqueIdentifier != "" {
 		name = input.UniqueIdentifier
 	}
+
+	// TODO - consider UpgradeCode here?
```

Comment:
Note to reviewer - this is  TODO for future consideration, not needed for this iteration